server/controllers/api/assessmentDayDataController/index.js

```diff
@@ -60,11 +60,22 @@ const AssessmentDayDataController = {
           }
         )
       } else {
-        await SiteMetadataModel.upsert(
-          appDb,
-          { participants: { $elemMatch: { participant } } },
-          { $set: { 'participants.$.synced': new Date() } }
-        )
+        const isParticipantInDocument = await SiteMetadataModel.findOne(appDb, {
+          participants: { $elemMatch: { participant } },
+        })
+        if (isParticipantInDocument) {
+          await SiteMetadataModel.upsert(
+            appDb,
+            { participants: { $elemMatch: { participant } } },
+            { $set: { 'participants.$.synced': new Date() } }
```

Comment:
Can we use $set or do we need $addField again?

Reply:
We can use $set here because it's with an update method, the $set that is lacking support is when we use it with an `aggregate()` method